SafeNeighbourhood/Pages/Index.cshtml

```diff
@@ -2,36 +2,63 @@
 @model IndexModel
 @{
     ViewData["Title"] = "Home page";
-    var welcomes = (QuickTypeVProperty.VacantProperty[]) ViewData["property"];
- 
+    //var welcomes = (QuickTypeVProperty.VacantProperty[]) ViewData["property"];
```

Comment:
Delete this line if it's not needed.  It's considered bad practice to commit commented code and push it to version control.  You can always go back in history to see what used to be there.